libs/chrome-tools/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Nuclia tools",
+  "description": "Upload web content to Nuclia search engine",
+  "version": "0.1",
```

Comment:
shouldn't it be semantic versioning with `major.minor.patch` format like `0.0.1`?

Reply:
Fixed